tests/Kiota.Builder.Tests/Plugins/PluginsGenerationServiceTests.cs

```diff
@@ -525,6 +525,64 @@ public async Task GeneratesManifestWithAdaptiveCardWithoutExtensionAsync()
 
     }
 
+    [Fact]
+    public async Task GeneratesManifestWithDefault200ResponseAsync()
+    {
+        var simpleDescriptionContent = @"openapi: 3.0.0
+info:
+  title: test
+  version: 1.0
+servers:
+  - url: http://localhost/
+    description: There's no place like home
+paths:
+  /test:
+    get:
+      description: description for test path
+      externalDocs:
+        description: external docs for test path
+        url: http://localhost/test
+      responses:
+        '400':
```

Comment:
Does this mean you are reintroducing the 4XX responses in this fix?

Reply:
No. the PR adds a default 200 response since the 4xx are intentionally omitted. 
The tests confirms that if it receives a yml file whose path only has a 400 response, we expect the 400 to be omitted and replaced by a default 200

Reply:
Got it. Thanks.